homeassistant/components/tolosauna/config_flow.py

```diff
@@ -15,6 +15,7 @@
 from homeassistant.data_entry_flow import FlowResult
 from homeassistant.helpers.device_registry import format_mac
 
+from ...helpers.typing import DiscoveryInfoType
```

Comment:
Don't use relative imports outside of this integration packaging.

Use `homeassistant.helpers.typing import DiscoveryInfoType` instead.